src/content/contributing/index.md

```diff
@@ -1,12 +1,313 @@
+- [Work on an open issue](https://github.com/ethereum/ethereum-org-website/issues)
+  _– Work we've identified that needs doing_
+- [Join the translations program ](https://ethereum.org/en/languages/#ethereum-org-translation-program)
+  _– Help us bring ethereum.org to new languages_
+  //I think this could be it's own page under contributing eventually
+- [Add community articles ](/en/contributing/adding-articles)
+  _– Add a helpful article to a relevant page_
+- [Add a product ](/en/contributing/adding-products/)
+  _– Add a dapp or wallet to a relevant page_
+- [Add developer tools ](/en/contributing/adding-developer-tools/)
+  _– Add a developer tool to a relevant page_
+- [Improve our research](https://www.notion.so/efdn/Ethereum-org-User-Persona-Memo-b44dc1e89152457a87ba872b0dfa366c)
+  _– Let us know your feedback on our research or contribute your own_
+- [Request a feature](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.md&title=)
+  _– Let us know about any ideas you have for a new feature or design_
+- [Create/edit content ](/en/contributing/#how-to-update-content)
+  _– Suggest new pages or makes tweaks to what's here already_
+
+_Any questions?_ 🤔 Reach out on our [Discord server](https://discord.gg/CetY6Y4)
+
+## How to work on ethereum.org {#how-to-update-content}
+
+Whether you're adding to the site, creating content or working on open issues, you'll need a [GitHub](https://github.com) account.
+
+All updates are made via the GitHub PR process. This means you create a local copy of the website, make your changes and request to merge in your changes.
+
+If you've not done this before, follow the instructions at the bottom of our [GitHub repository](https://github.com/ethereum/ethereum-org-website).
+
+You don't need permission to work on anything, but it's always best to let us know what you're planning to do. You can do this by:
+
+- Commenting on an issue or PR in [GitHub](https://github.com/ethereum/ethereum-org-website)
+- Messaging on [Discord server](https://discord.gg/CetY6Y4)
+
+Before contributing , make sure you're familiar with:
+
+- the evolving [vision of ethereum.org](/about/)
+- our [design principles](/en/contributing/design-principles/)
+
+## How decisions about the site are made
+
+Decisions about individual PRs, design evolution and major upgrades are made by a team from across the Ethereum ecosystem. This team includes project managers, developers, designers, marketing and communications, and subject matter experts. Community input informs every decision: so please raise questions in issues, submit PRs, or contact the team:
+
+- [website@ethereum.org][website@ethereum.org]
+- [@ethdotorg](https://twitter.com/ethdotorg)
+- [Discord server](https://discord.gg/CetY6Y4)
+
+## Contributors
+
+Thanks to everyone who has contributed so far!
+
+<table>
```

Comment:
This is great for a v1 but we should create an automated process that copies over this content (i.e. from [the .all-contributorssrc file](https://github.com/ethereum/ethereum-org-website/blob/dev/.all-contributorsrc)) so that it's updating as new contributions are made.

Reply:
I created this automated process 😄